conf/standalone.conf

```diff
@@ -590,7 +590,7 @@ managedLedgerDefaultAckQuorum=1
 
 # How frequently to flush the cursor positions that were accumulated due to rate limiting. (seconds).
 # Default is 60 seconds
-managedLedgerCursorPositionFlushSeconds = 60
+managedLedgerCursorPositionFlushSeconds=60
```

Comment:
@BewareMyPower do these spaces matter?

Reply:
yes, that's

Reply:
Yes, that's right

Reply:
> @BewareMyPower do these spaces matter?
Yes, that's right

Reply:
I think it doesn't matter, at least on MacOS.